src/Cake.Npm/NpmSettingsExtensions.cs

```diff
@@ -48,5 +48,51 @@ public static NpmSettings FromPath(this NpmSettings settings, DirectoryPath path
 
             return settings;
         }
+
+        /// <summary>
+        /// Sets the StandardError-Action
+        /// </summary>
+        /// <param name="settings">The settings.</param>
+        /// <param name="standardErrorAction">The StandardError-Action.</param>
+        /// <returns>The <paramref name="settings"/> instance with <see cref="Core.Tooling.ToolSettings.WorkingDirectory"/> set to <paramref name="path"/>.</returns>
+        public static NpmSettings WithStandardError(this NpmSettings settings, Action<string> standardErrorAction)
```

Comment:
I would suggest to follow [naming in Cake.Core](https://github.com/cake-build/cake/blob/develop/src/Cake.Core/Extensions/ProcessSettingsExtensions.cs#L69): `SetStandardErrorHandler`

Reply:
changed.